net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml

```diff
@@ -1,12 +1,16 @@
 <model>
     <mount>//OPNsense/ntopng/general</mount>
     <description>ntopng configuration</description>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
     <items>
         <enabled type="BooleanField">
             <default>0</default>
             <Required>Y</Required>
         </enabled>
+        <community type="BooleanField">
+            <default>0</default>
```

Comment:
Why not default to enabled? I even thought about using it unconditionally?

Reply:
Becasue for me it also works without it, why changing and may breaking running setups.

Reply:
I agree with mimugmail.
By default ntopng 4.3 starts in Enterprise edition for 10 minutes and then downscales to community edition. Forcing it to community edition with --community makes the interface consistent for non-licensed clients.
Clients that want to try Enterprise features - or even purchase it - should have an option to run it without this switch.

Reply:
It doesn’t really answer the need for sane default question but we can take it to the other PR.